css/class-naming-conventions/readme.md

```diff
@@ -43,23 +44,23 @@ Our convention (which we call CSM or Component, Sub-Component, Modifier) uses [B
 
 This example may seem confusing at first but if we break down each of the selectors that we have, it begins to make more sense.
 
-`.c-blog` This is a component. It describes a high level module or component. In this instance, it describes the container for all of our blog posts.
+`c-blog` This is a component. It describes a high level module or component. In this instance, it describes the container for all of our blog posts.
```

Comment:
"This is a high-level component" might be enough here. What do you think about avoiding the word "module" so that our language stays consistent? I know components were sometimes called modules in old projects, but we don't do that anymore. ✂️